tests/planning_poker/test_admin.py

```diff
@@ -1,6 +1,8 @@
+from django.contrib.admin import site
 import pytest
```

Comment:
`import` lines should always come before `from ... import` lines in each group of imports. You could probably just run isort to fix this everywhere.